### DISCUSSION TITLE
What is an interactive CLI way to set/change a theme?

### DISCUSSION THREAD

**Discussion by 4l1fe** [JanDeDobbeleer/oh-my-posh#4708] [General]
As of now, i go to the `zsh` config and change a theme path manually with further reloading a terminal.

Tha's obviously unhandy way to do. 

Is there something like ?
```
oh-my-posh theme set bubbles  
```

**Comment by JanDeDobbeleer** [comment]
Yes, you can change the value of the environment variable `POSH_THEME` to the path of your preferred configuration.

**Comment by 4l1fe** [comment]
It works, thx
